resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/resources/usecase/SnapshotTest.java

```diff
@@ -64,7 +65,7 @@ public void test2() {
 		test.testSnapshotWorkspace();
 	}
 
-	public void test3() {
+	public void test3() throws CoreException {
 		if (skipTest()) {
```

Comment:
Use assumptions to skip this test visibly. It should be possible with JUnit4. If not, check whether or not you need to annotate the class with `@RunWith(JUnit4.class)`

Reply:
These are `WorkspaceSessionTests` that still rely on JUnit 3. It is not possible to simply attach an `@RunWith(JUnit4.class)`, since they are executed by `WorkspaceSessionTestSuites` with some custom behavior.

Reply:
👍  Dito (Tests are still JUnit3)